libtensor/core/allocator.h

```diff
@@ -29,7 +29,7 @@ class allocator_wrapper_i {
     virtual void shutdown() = 0;
     virtual size_t get_block_size(size_t sz) = 0;
     virtual pointer_type allocate(size_t sz) = 0;
-    virtual void deallocate(const pointer_type &p) throw () = 0;
```

Comment:
missed one :smile:

Reply:
Dang it, these things are everywhere! There is kind of a lot of special cases to the regex...